lib/elixir/test/elixir/list_test.exs

```diff
@@ -22,6 +22,11 @@ defmodule ListTest do
     assert (&[&1, &2 | &3]).(1, 2, 3) == [1, 2 | 3]
   end
 
+  test "delete/2" do
+    assert List.delete([:a, :b, :c], :a) == [:b, :c]
```

Comment:
These tests are already generated from documentation examples as doctests (more detailed information on how doctests work can be found here: https://hexdocs.pm/ex_unit/ExUnit.DocTest.html)

Reply:
Although originally we used the doctests for coverage, we tend to not rely on doctests for tests, since they are about documentation first and foremost. :)

Reply:
:+1: